packages/backend-tasks/src/tasks/PluginTaskSchedulerImpl.test.ts

```diff
@@ -133,14 +141,19 @@ describe('PluginTaskManagerImpl', () => {
       async databaseId => {
         const { manager } = await init(databaseId);
 
-        const fn = jest.fn();
+        const { promise, resolve } = defer();
+
         await manager.scheduleTask({
           id: 'task1',
           timeout: Duration.fromMillis(5000),
           frequency: Duration.fromObject({ years: 1 }),
-          fn,
+          fn: async () => {
+            resolve();
+            await new Promise(r => setTimeout(r, 20000));
```

Comment:
Why is it needed?

Reply:
The test wants the task to be actively running, and then trying to cause a conflict while it's executing. So the task needs to block for a while so that the conflict has time to happen before it ends.

Reply:
The test wants the task to be actively running, and then trying to cause a conflict while it's executing. So the task needs to block for a while so that the conflict has time to happen before it ends.